examples/bingo/README.md

```diff
@@ -30,7 +30,7 @@ Create file `secret.json`:
 }
 ```
 
-Once the app is installed, copy the _OAuth Access Token_ to the `"oauthToken"` field value in `secret.json`. 
+Once the app is installed, go back to app's Settings -> Basic Information -> App Credentials where client secret and versification token can be found. Copy the _OAuth Access Token_ to the `"oauthToken"` field value in `secret.json`.
```

Comment:
_Verification Token_ can be found in Settings -> Basic Information
_OAuth Access Token_ can be found in Features -> OAuth & Permissions

Reply:
I just updated it